src/js/Content/Features/Store/App/FDLCCheckboxes.js

```diff
@@ -20,6 +20,18 @@ export default class FDLCCheckboxes extends Feature {
             const label = document.createElement("label");
             label.classList.add("es_dlc_label");
 
+            // Add dsinfo to label for use with select/unselect all
+            if (dlcRow.classList.contains("ds_wishlist")) {
+                label.classList.add("es_dlc_wishlist");
+            } else if (dlcRow.classList.contains("ds_owned")) {
+                label.classList.add("es_dlc_owned");
+            }
+
+            // Toggle dsinfo when adding/removing wishlist via ds_options dropdown
+            new MutationObserver(() => {
+                label.classList.toggle("es_dlc_wishlist", dlcRow.classList.contains("ds_wishlist"));
+            }).observe(dlcRow, {"attributeFilter": ["class"]});
```

Comment:
We're potentially creating a lot of `MutationObserver`s here, maybe it would be better to collect this data when the actual click on "Select Wishlisted DLC" is happening

Reply:
Changed it to a single `MutationObserver` that observes `.gameDlcBlocks` instead.